test/e2e/scenarios/namespaceselector.go

```diff
@@ -0,0 +1,14 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Namespace Selector based Network Policies for ingress between clusters", func() {
```

Comment:
As discussed, can you please link each of the scenario with the use-case document.